src/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from './App';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7272556)
```suggestion
import App from "./App";
```